packages/reactotron-server/src/server/apollo/resolvers/executedCommands.ts

```diff
@@ -0,0 +1,46 @@
+import { Resolver, Query, Subscription, Root, Mutation, Arg } from "type-graphql"
+import * as GraphQLJSON from "graphql-type-json"
+
+import { messaging, MessageTypes } from "../../messaging"
+import { ExecutedCommand } from "../../schema"
+import { executedCommands } from "../../datastore"
+import { createReactotronServer } from "../../reactotron"
+
+@Resolver()
+export class ExecutedCommandsResolver {
+  @Query(() => [ExecutedCommand])
+  executedCommands() {
+    return executedCommands.all()
+  }
+
+  @Mutation()
+  executeCommand(
+    @Arg("type") type: string,
+    @Arg("payload", () => GraphQLJSON) payload: object,
+  ): boolean {
+    const newCommand = {
+      type,
+      date: new Date(),
+      payload,
+    }
+
+    executedCommands.addCommand(newCommand)
+    messaging.publish(MessageTypes.EXECUTED_COMMAND_ADDED, newCommand)
+
+    // Welcome to the hack zone. Where everything is permitted as long as it works.
+    // TODO: Better name? Put this into a singleton that we can import and use instead of this function? Better everything? something? ANYTHING?
+    const reactotronServer = createReactotronServer()
```

Comment:
I want to change how I handle the reactotron server but this serves the purpose for now. Note that this function returns a singleton so we are not actually re-creating the server every time this is called despite the name of the function.

Reply:
I've made a similar comment at the top about Apollo server. A standard for the project may help.